src/components/views/elements/Field.js

```diff
@@ -57,7 +60,22 @@ export default class Field extends React.PureComponent {
 
         const fieldInput = React.createElement(inputElement, inputProps, children);
 
-        return <div className={`mx_Field mx_Field_${inputElement}`}>
+        let prefixContainer = null;
+        if (prefix) {
+            prefixContainer = <span className="mx_Field_prefix">
+                {prefix}
```

Comment:
nit: this should fit on a single line

Reply:
Thanks, fixed in https://github.com/matrix-org/matrix-react-sdk/pull/2749/commits/35ee9c9ac730b4f1c56317de2d99d096c1f544f7.